saviSimulation/savi/jason_processing/WifiAntenna.java

```diff
@@ -74,4 +73,17 @@ public void receiveMessage(String msg) {
 		communicator.receiveMessage(msg);
 	}
 
+	// takes probability parameter between 0 and 1 
+	protected boolean isWifiFailing(double probability, int seed) {
+		Random rand = new Random();
+		if(seed != -1) {
+			rand = new Random(seed);
```

Comment:
This will not be random because there is a new Random object every time. Since the seed is locked it will always use the first random number.

Reply:
Fixed

Reply:
This has not been fixed.